docs/network/default_cni_plugin.md

```diff
@@ -0,0 +1,156 @@
+nmcli con up ovs-if-br-ex
+nmcli con mod ovs-if-phys0 connection.autoconnect yes
+nmcli con mod ovs-if-br-ex connection.autoconnect yes
+```
+
+Replace `enp1s0` with the network interface name where node IP address is assigned to. <br>
+Replace `1500` with the actual MTU on the network interface. <br>
+
+**NOTE:* Copy the above NetworkManager command in a script and execute them at once. <br>
+**NOTE:* Execution of the above commands will cause transient network disconnection from the node IP. <br>
+
+[comment]: # (TODO: replace OVS commands with nmcli which can be easily installed under /etc)
+
+## Network Features
+
+A wide range of networking features are available with MicroShift and ovn-kubernetes, including but not limited to:
+
+* Network policy
+* Dynamic node IP
+* Custom gateway interface
+* Second gateway interface
+
+### Network Policy
+
+Network Policy restricts network traffic to and/or from kubernetes pods.
+The ovn-kubernetes implementation of network policy supports pod, namespace and ipBlock based identifiers as well as Ingress and Egress isolation types.
+See [ovn-kubernetes network policy](https://github.com/ovn-org/ovn-kubernetes/blob/master/docs/network-policy.md) doc for detailed design and configurations.
+
+### Dynamic node IP
+
+MicroShift is able to detect node IP change and restarts itself to take in the new IP address.
+Upon restarting, it recreates ovnkube-master daemonset with updated IP address in openshift-ovn-kubernetes namespace.
+
+### Custom gateway interface
+
+microshift-ovs-init.service is able to use user specified host interface for cluster network.
+This is done by specifying the `gatewayInterface` in the CNI config file `/etc/microshift/ovn.yaml`.
+The specified interface will be added in OVS bridge `br-ex` which acts as gateway bridge for ovn-kubernetes CNI network.
+
+### Second gateway interface
+
+microshift-ovs-init.service is able to setup one additional host interface for cluster ingress/egress traffic.
+This is done by specifying the `externalGatewayInterface` in the CNI config file `/etc/microshift/ovn.yaml`.
+The external gateway interface will be added in a second OVS bridge `br-ex1`. Cluster pod traffic destinated to additional host subnet will be routed through `br-ex1`.
+
+## Known Issues
+
+* Firewall reload flushes iptable rules
+
+ovn-kubernetes makes use of iptable rules for some traffic flows (such as nodePort service), these iptable rules are generated and insertd by ovn-kubernetes (ovnkube-master container), but can be removed by reloading firewall rules, which in turn breaks the traffic flows. To avoid such situation, make sure to execute firewall commands before starting ovn-kubernetes pods. If firewall commands have to be executed after ovn-kubernetes pods have started, manually restart the ovnkube-master pod to trigger reinsertion of ovn-kubernetes iptable rules. See section [NodePort Service](#external-to-nodeportservice) for details on the iptable rules.
```

Comment:
```suggestion
ovn-kubernetes makes use of iptable rules for some traffic flows (such as nodePort service), these iptable rules are generated and inserted by ovn-kubernetes (ovnkube-master container), but can be removed by reloading firewall rules, which in turn breaks the traffic flows. To avoid such situation, make sure to execute firewall commands before starting ovn-kubernetes pods. If firewall commands have to be executed after ovn-kubernetes pods have started, manually restart the ovnkube-master pod to trigger the reinsertion of ovn-kubernetes iptable rules. See section [NodePort Service](#external-to-nodeportservice) for details on the iptable rules.
```